docs/spfx/dynamic-loading.md

```diff
@@ -81,7 +81,7 @@ This can be achieved by modifying **config/config.json** so that each web part i
 
 ## Analyze your bundle
 
-In order to get an overview of where to start,  optimize, see the article [Optimize SharePoint Framework builds for production, which illustrates how you can get a visual map of which part of your code or third-party library takes up space in your bundle.
+In order to get an overview of where to start,  optimize, see the article [Optimize SharePoint Framework builds for production](https://docs.microsoft.com/sharepoint/dev/spfx/toolchain/optimize-builds-for-production), which illustrates how you can get a visual map of which part of your code or third-party library takes up space in your bundle.

```

Comment:
thanks for the submission, but can you please (1) use a relative link & (2) update the `ms.date` in the frontmatter to reflect when the doc was updated?

Reply:
Sure,
(1) the previous version of an article used absolute link so i just took it from there :)
(2) I was not aware of that requirement
I will correct it immediately